checks.d/snmp.py

```diff
@@ -238,6 +238,8 @@ def check(self, instance):
         ip_address = instance["ip_address"]
         table_oids = []
         raw_oids = []
+        timeout = instance.get('timeout', self.DEFAULT_TIMEOUT)
+        retries = instance.get('retries', self.DEFAULT_RETRIES)
```

Comment:
It's unclear from what I've just read of the pysnmp doc (and the code is horrible to read) if a string value is OK here, so it might be a good idea to cast these values as integers?